desktop-exporter/app/components/waterfall-view/duration-bar.tsx

```diff
@@ -12,18 +17,35 @@ type DurationBarProps = {
 };
 
 export function DurationBar(props: DurationBarProps) {
+  const ref = useRef(null);
+  const size = useSize(ref);
+  const labelWidth = 80;
+
   let durationBarColour = useColorModeValue("cyan.800", "cyan.700");
+  let labelTextColour = useColorModeValue("blackAlpha.800", "white");
 
   let { traceStartTimeNS, traceDurationNS } = props.traceTimeAttributes;
   let spanStartTimeNs = getNsFromString(props.spanStartTimestamp);
   let spanEndTimeNs = getNsFromString(props.spanEndTimestamp);
 
-  let offsetStart = Math.floor(
+  let barOffset = Math.floor(
```

Comment:
comments that these are %-based values would likely be helpful for future readers, or rename to `barOffsetPercent`